### ISSUE TITLE
Corpse Loot Bug

### ISSUE THREAD

**Issue by Weuyn** [Stormforge-gg/Bugtracker-Fordring#1751]
**Description**: 
Uncommons+ that are rolled for on corpses block all other loot from being looted off of the body
**Current behaviour**:
You kill a guy in a party
He drops a green item and 2 grey items
grey items cannot be looted, 'item is still being rolled for'
The party rolls for the green item, someone receives it
The grey items remain unlootable on the body, 'item is still being rolled for'
**Expected behaviour**:

**Comment by Vishiswaz** [comment]
Nope, the items you are seeing belong to someone else. The bug here is that common items that belong to someone else should not be viewable on a corpse with a green+ quality item on it that is being rolled for.

**Comment by Weuyn** [comment]
that could also be the case
